R/annotation-data-helper.R

```diff
@@ -50,7 +50,7 @@ getFirstExonRanges <- function(exonRangesByTx.unlist) {
 }
 
```

Comment:
Code is optimized taking into account metadata not stored in the reduced promoter annotation object.
Code that is used to generate such metadata is removed.